src/main/kotlin/com/bridgecrew/initialization/InitializationService.kt

```diff
@@ -44,6 +45,9 @@ class InitializationService(private val project: Project) {
         }
 
         setSelectedCheckovService(DockerCheckovScanCommandsService(project))
+        if(!output.lowercase().trim().contains("pulling from bridgecrew/checkov")){
```

Comment:
we're sure this is a 1:1 relationship with running on docker (no chance it's being printed in the regular or other form right?)

Reply:
yes